playbook.yml

```diff
@@ -45,10 +45,34 @@
         - supervisor
         - python-mysqldb
```

Comment:
I've been wanting to turn "with_items" into a variable...

Let's turn this package list into a variable, then alter the list if it's NGINX, instead of having duplicate sections in the playbook.yml.